lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -503,7 +498,8 @@ void OpenCLFunction::executeConvolution(const OCLConvolutionInst *CC) {
 
   // Generate a tailor-made convolution kernel using the provided options based
   // on the parameters of the current convolution.
-  auto &programName = isQuantized ? FWD_CONV_QUANTIZED_CODE : FWD_CONV_CODE;
+  const char *programName = isQuantized ? (const char *)FWD_CONV_QUANTIZED_CODE
```

Comment:
Can you use C++-style casts here, e.g. `static_cast`?

Reply:
reinterpret_cast<const char *>(SHADER_CODE) instead